test/versioned/hapi/hapi-post-18/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "hapi-v17-tests",
+  "version": "0.0.0",
+  "private": true,
+  "tests": [
+    {
+      "engines": {
+        "node": ">=8.9"
+      },
+      "dependencies": {
+        "ejs": "2.5.5",
+        "@hapi/hapi": ">=18.3.1",
```

Comment:
Should this be loosened to test older versions of v18? Maybe something like `"^18.3.1"`?

Reply:
Ah, will change to ^18.0.0.